app/assets/javascripts/geoblacklight/templates/index_map_info.hbs

```diff
@@ -12,26 +14,26 @@
         <img src="{{thumbnailUrl}}">
       {{/if}}
     {{/if}}
-    <dl>
+    <dl class="row dl-invert document-metadata">
       {{#if websiteUrl}}
-        <dt>Website</dt>
-        <dd><a href="{{websiteUrl}}">{{websiteUrl}}</a></dd>
+        <dt class="blacklight-dc_creator_sm col-sm-3">Website:</dt>
```

Comment:
Could we use a more generic class here? This seems to be a specific class focused on `dc_creator_sm`